src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -2,25 +2,27 @@ package uk.gov.hmcts.contino;
 
 class YarnBuilder extends AbstractBuilder {
 
+  private static final String INSTALL_CHECK_FILE = '.yarn_dependencies_installed'
+
   YarnBuilder(steps) {
     super(steps)
   }
 
   def build() {
-    yarn("--mutex network install --frozen-lockfile")
-    yarn("lint")
+    runYarn("--mutex network install --frozen-lockfile")
```

Comment:
shouldn't all these methods call `yarn`?

you could then get rid of this line as it will get invoked by the lint call